stdlib/argparse.pyi

```diff
@@ -357,7 +357,18 @@ class Action(_AttributeHolder):
 
 if sys.version_info >= (3, 12):
     class BooleanOptionalAction(Action):
-        if sys.version_info >= (3, 13):
+        if sys.version_info >= (3, 14):
+            def __init__(
+                self,
+                option_strings: Sequence[str],
+                dest: str,
+                default: bool | None = None,
+                *,
```

Comment:
The arguments are not keyword-only. It doesn't make much sense to pass them positionally, but it works, so we might as well reflect it in the stub.
```suggestion
```